v2/cpu.go

```diff
@@ -19,8 +19,6 @@ package v2
 type CPU struct {
 	Weight *uint64
 	Max    *uint64
-	Cpus   string
```

Comment:
where has it gone?

Reply:
According to documentation https://www.kernel.org/doc/Documentation/cgroup-v2.txt
`cpuset.cpus and cpuset.mems`, that caused errors, since there is no such resources in v2

Reply:
@crosbymichael PTAL

Reply:
It exists if you toggle `cpuset` controller